packages/pipeline-editor/src/PipelineEditor/index.tsx

```diff
@@ -524,9 +526,16 @@ const PipelineEditor = forwardRef(
           if (filenameRef) {
             payload =
               e.targetObject?.app_data?.component_parameters?.[filenameRef];
+          } else {
+            type = "openComponentDef";
+            payload =
+              controller.current
+                .getAllPaletteNodes()
+                .find((n) => n.op === e.targetObject.op)?.id ??
+              e.targetObject.op;
```

Comment:
Why did you choose this fallback?

Reply:
I mentioned it in the other PR

> have re-added my fallback code for when a component is not in the palette. The fallback will pass the component op value for the node when it can't find the component in the palette to look up it's id (note that our current airflow and kfp examples mirror op and id but generic components do not). This at least makes the backend error more readable.

Reply:
Ah just found that and was about to mark this as resolved 😅